### ISSUE TITLE
Mod suggestion

### ISSUE THREAD

**Issue by Tanmay850** [xJon/The-1.12.2-Pack#487]
put:
gravity gun
project e 
darconic evolution 
bewitchment 
agricraft
immersive engerneering
cyclic 
baubly heart canisters  all great and useful mod

**Comment by superkitten12** [comment]
I agree with the heart canisters my 3 health bars of runic shielding isnt enough, especially when im taking 0 damage from anything with my power armor

**Comment by n2galaxy** [comment]
Half of these are overpowered and silly. Draconic in particular is one excellent way to instantly ruin a modpack.

**Comment by MooneyMC** [comment]
Immersive Engineering is a really cool functional machines mod, so there's probably a reason why it isn't already in this pack.
